README.md

```diff
@@ -34,6 +37,18 @@ database.  If that is successful you can run the app server with:
 
     $ foreman start
 
+### Getting Started (docker)
+
+* Install [Docker Compose](https://docs.docker.com/compose/install/)
+  * On non-linux systems this will involve installing Docker Toolbox and
+    starting a Docker Machine VM.
+* Build/download the container images: `docker-compose build`
+* Start the app: `docker-compose up -d`
+* Setup the database: `docker-compose run --rm web rake db:setup RAILS_ENV=development`
+
+See the Docker Compose [docs](https://docs.docker.com/compose/install/) for
+more information.
+
```

Comment:
Can we also add a note about how to stop docker? I had to Google it a few times before finding the command `docker-machine stop`. The VirtualBox image was using over 1GB of RAM and I wanted to shut it down when not in use.